server/tests/backends/fixtures/custom/simple.json

```diff
@@ -1,67 +1,121 @@
 {
-  "exclude": [
-    "mysql",
-    "pandas",
-    "postgres",
-    "mongo"
-  ],
-  "step": {
-    "name": "customsql",
-    "pipeline": [
-      {
-        "name": "customsql",
-        "query": "SELECT AGE FROM ##PREVIOUS_STEP## WHERE AGE > 0;"
-      }
-    ]
-  },
-  "input": {
-    "schema": {
-      "fields": [
-        {
-          "name": "NAME",
-          "type": "string"
-        },
-        {
-          "name": "AGE",
-          "type": "integer"
-        },
-        {
-          "name": "SCORE",
-          "type": "integer"
-        }
-      ],
-      "pandas_version": "0.20.0"
+    "exclude": [
+        "snowflake",
+        "postgres",
+        "mysql",
+        "pandas"
+    ],
+    "step": {
+        "name": "custom",
+        "pipeline": [
+            {
+                "name": "custom",
+                "query": "[{\"$project\": {\"COMPANY\": \"$COMPANY\", \"COUNTRY\": \"$COUNTRY\", \"_vqbToUnpivot\": {\"$objectToArray\": {\"CURRENCY\": {\"$ifNull\": [\"$CURRENCY\", null]}, \"PROVIDER\": {\"$ifNull\": [\"$PROVIDER\", null]}}}}}, {\"$unwind\": \"$_vqbToUnpivot\"}, {\"$project\": {\"COMPANY\": \"$COMPANY\", \"COUNTRY\": \"$COUNTRY\", \"KPI\": \"$_vqbToUnpivot.k\", \"VALUE\": \"$_vqbToUnpivot.v\"}}]"
```

Comment:
is it not better to replace from `\"` -> `'` ?

Reply:
The fixture and the query need to be valid JSON, so we can't use single quotes... But I agree, it's kinda painful to read :sweat: